langchain/vectorstores/chroma.py

```diff
@@ -98,6 +109,7 @@ def __init__(
             else None,
             metadata=collection_metadata,
         )
+        self.override_relevance_score_fn = relevance_score_fn
```

Comment:
@raymond-yuan worth having a look at this related PR:
https://github.com/hwchase17/langchain/pull/7084